app/src/main/java/org/fossasia/openevent/general/auth/EditProfileFragment.kt

```diff
@@ -74,7 +75,7 @@ class EditProfileFragment : Fragment() {
         editProfileViewModel.progress
             .nonNull()
             .observe(this, Observer {
-                rootView.progressBar.isVisible = it
+                rootView.epProgressBar.isVisible = it
```

Comment:
Why `epProgressBar` ?

Reply:
earlier it was showing some name ambiguity with progressbar of profile fragment but now it is fixed, I will change it